packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterApiRequest.kt

```diff
@@ -30,16 +30,17 @@ object FlutterApiRequest {
     private const val BODY_KEY = "body"
     private const val QUERY_PARAM_KEY = "queryParameters"
     private const val HEADERS_KEY = "headers"
+    private val TABS_OUTSIDE_QUOTES_PATTERN = Regex("\t(?=(?:[^'\"`]*(['\"`])[^'\"`]*\\1)*[^'\"`]*\$)")
 
     // ====== Rest API ======
     fun getCancelToken(request: Map<String, Any>): String {
         try {
             return request[CANCEL_TOKEN_KEY] as String
         } catch (cause: Exception) {
             throw AmplifyException(
-                "The cancelToken request argument was not passed as a String",
```

Comment:
Lots of formatting changes here, go straight to line 116 for interesting bit.

Reply:
Can also just reference https://github.com/aws-amplify/amplify-flutter/pull/1178/files?diff=split&w=1